### PR TITLE
Adding tutorial on how to clean and manage AB1 files from Sanger sequencing

### PR DESCRIPTION
**Pull request by ColineRoyaux**
Hi !

A new tutorial that I made based on the cleaning workflow I created for managing and cleaning Sanger sequencing data from my PhD, it has been asked by many teamates of mine :) 

I loaded it in the "ecology" folder but I'm not sure it is the best place ! Scientists other than ecologists use Sanger sequencing, but I don't know in which section it fits best... Help? ;)

Hope it looks good for you !

### CODE REVIEW

**Review by hexylena — COMMENTED** [review]
I think it could fit in `sequence-analysis`? It's a bit of an odd topic but that's where we direct people for NGS questions, maybe also for original generation sequencing too :) 

This is super cool to see! I worked with ab1 some years back, really great to see others doing it too :)

File: topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,530 @@
...
+
+## Unzip data files
+
+> <hands-on-title> Unzip </hands-on-title>
+>
+> 1. {% tool [Unzip](toolshed.g2.bx.psu.edu/repos/imgteam/unzip/unzip/6.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"input_file"*: `AOPEP_and_CHD8_sequences_20220907.zip?download=1`
+>    - *"Extract single file"*: `All files`
+>
+> > ### {% icon question %} Question
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.md:173]:
```suggestion
> > ### <question-title></question-title>
```

File: topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,530 @@
...
+> > How many files is there in the ZIP archive ?
+> >
+> > > ### {% icon solution %} Solution
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.md:176]:
```suggestion
> > > <solution-title></solution-title>
```

File: topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,530 @@
...
+
+## Merge and align consensus sequences file and primer files
+
+> <hands-on-title> Merge and format consensus sequences + primers file </hands-on-title>
+>
+> 1. {% tool [Merge.files](toolshed.g2.bx.psu.edu/repos/iuc/mothur_merge_files/mothur_merge_files/1.39.5.0) %} with the following parameters:
+>    - *"Merge"*: `fasta files`
+>        - {% icon param-files %} *"inputs - fasta"*: `consensus sequences` (output of **Merge.files** {% icon tool %}), `Reverse primer` (output of **Reverse-Complement** {% icon tool %}), `Forward primer` (output of **Degap.seqs** {% icon tool %})
+> 
+>    {% snippet tools_select_multiple_datasets.md %}
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.md:377]:
```suggestion
>    {% snippet faqs/galaxy/tools_select_multiple_datasets.md %}
```

File: topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,530 @@
...
+
+It is a good practice to proceed to such checks, its permits to make sure the sequencing went as planned and your samples haven't been contaminated. 
+
+# Conclusion
+
+We successfully cleaned AB1 sequence files !
+
+# AOPEP Sanger files
+
+The history following the same steps but for AOPEP marker files is available [here](https://usegalaxy.fr/u/coline_royaux/h/tuto-sanger-w-aopep)
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.md:530]:
```suggestion
The history following the same steps but for [AOPEP marker files is available](https://usegalaxy.fr/u/coline_royaux/h/tuto-sanger-w-aopep)
```

File: topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.md [hunk 5]
```diff
@@ -0,0 +1,530 @@
...
+
+## Unzip data files
+
+> <hands-on-title> Unzip </hands-on-title>
+>
+> 1. {% tool [Unzip](toolshed.g2.bx.psu.edu/repos/imgteam/unzip/unzip/6.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"input_file"*: `AOPEP_and_CHD8_sequences_20220907.zip?download=1`
+>    - *"Extract single file"*: `All files`
+>
+> > <question-title>Question</question-title>
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.md:173]:
```suggestion
> > <question-title></question-title>
```

the `qusetion` is added automatically if it is empty :)

File: topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.md [hunk 6]
```diff
@@ -0,0 +1,530 @@
...
+> > How many files is there in the ZIP archive ?
+> >
+> > > <solution-title>Solution</solution-title>
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.md:176]:
```suggestion
> > > <solution-title></solution-title>
```

**Review by hexylena — COMMENTED** [review]
galaxy-barchat - this error is my fault, sorry.

**Comment by ColineRoyaux** [comment]
Thank you for your help @hexylena ! It seems there is an issue with the "rerun a tool" snippet but I can't find what is wrong ...

File: topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.md [hunk 7]
```diff
@@ -0,0 +1,530 @@
...
+time_estimation: 1H
+key_points:
+- Check your data and results for mistakes afterward !!!! This procedure is useful but not perfect
+contributors:
+- colineroyaux
+
+---
+
+
+# Introduction
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.md:20]:
```suggestion
```

not necessary to include this heading

File: topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.md [hunk 8]
```diff
@@ -0,0 +1,530 @@
...
+>
+> 1. {% tool [Filter FASTA](toolshed.g2.bx.psu.edu/repos/galaxyp/filter_by_fasta_ids/filter_by_fasta_ids/2.3) %} with the following parameters:
+>    - {% icon param-file %} *"FASTA sequences"*: `Primer file`
+>    - *"Criteria for filtering on the headers"*: `Regular expression on the headers`
+>        - *"Regular expression pattern the header should match"*: `Reverse_CHD8`
+>    - Add tags "#Primer" and "#Reverse"
+>      
+>    {% snippet faqs/galaxy/datasets_add_tag.md %}
+>
+>  {% snippet faqs/galaxy/tools_rerun.md %}
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.md:132]:
```suggestion
>    {% snippet faqs/galaxy/tools_rerun.md %}
```

i'm not sure

File: topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.md [hunk 9]
```diff
@@ -0,0 +1,530 @@
...
+>            - *"Find Regex"*: `^[A-Za-z0-9_-]+F$`
+>            - *"Replacement"*: ``
+>        - {% icon param-repeat %} *"Insert Check"*
+>            - *"Find Regex"*: `^[A-Za-z0-9_-]+AOPEP[A-Za-z0-9_-]+$`
+>            - *"Replacement"*: ``
+>    - Tag output with "#Reverse"
+>
+>    {% snippet faqs/galaxy/datasets_add_tag.md %}
+>
+>  {% snippet faqs/galaxy/tools_rerun.md %}
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.md:211]:
```suggestion
>    {% snippet faqs/galaxy/tools_rerun.md %}
```

File: topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.md [hunk 10]
```diff
@@ -0,0 +1,530 @@
...
+
+## Convert AB1 sequence files to FASTQ and trim low-quality ends
+
+In Sanger sequencing, ends tend to be of low trust levels (each nucleotide has a quality score reflecting this trust level), it is important to delete these sections of the sequences to ensure wrong nucleotides aren't introduced in the sequences. 
+
+> <hands-on-title> AB1 to FASTQ files and trim low quality ends </hands-on-title>
+> 
+> Do these steps twice !! We have Froward and antisense (reverse) sequence data collections, do these steps starting with each "(filtered)" data collections, this could help:
+> 
+>  {% snippet faqs/galaxy/tools_rerun.md %}
```
Comment (hexylena) [inline topics/sequence-analysis/tutorials/Manage_AB1_Sanger/tutorial.md:259]:
```suggestion
> {% snippet faqs/galaxy/tools_rerun.md %}
```

**Review by hexylena — APPROVED** [review]

**Comment by hexylena** [comment]
Thanks @ColineRoyaux !

**Pull request merged**